community/good-member/README.md

```diff
@@ -0,0 +1,133 @@
+
+That means that the majority of discussions are happening between two people who want the best for each and also for Exercism.
+
+Sometimes, when disagreements happen its easy to jump to the conclusion that the other person is being difficult, or not listening, or not understanding your point.
+It's easy to imagine they're sitting there feeling smug, polishing their ego, while they rile you up with their wrong opinions.
+In reality, they're much more likely to be trying to communicate something that they feel is important and not finding a way to do that in a way that is clear to others.
+
+When we step back and presume each other are **tying** to be kind, and trying to do the right things, and trying to be clear, it makes it a lot easier for us to loosen the grip on our own egos and **actively** try to listen to the other person's point.
+
+Which brings us on to...
+
+## Being wrong is awesome
+
+Learning that you're wrong about something is the best thing that can happen to you, because you're now less wrong than you were before.
+If you're someone that cares about being scientific in your thinking, in being correct and knowledge, then discovering you're wrong is the most exciting thing possible.
+
+So when you go into a discussion on Exercism, see it as an opportunity not to prove you're right but to discover where you might be wrong, and in doing so become more correct than you were before.
+
+Adam Grant wrote a [whole book on this](https://www.adamgrant.net/book/think-again/).
+In it he quotes Ray Dalio saying "If you don't look back at yourself and think, 'Wow how sutpid I was a year ago', then you must not have learned much in the last year".
+
+## You don't know what you don't know
+
+Exercism is built on the tenant that we don't know what we don't know.
+The whole platform works around mentors pointing out knowledge gaps that you didn't know you have.
+And that works incredibly when both sides are open to knew information (very often its the mentors that realise they didn't know something when seeing new code from a student!)
+
+It's ease to remember that in the context of being a student in a mentoring session, but it's just as important everywhere else within Exercism.
+When you think something sucks about the website, start by asking yourself what it is that you might not know — a decision that looks ridiculous to you might indicate that your knowledge is missing something just as much as it might indicate that the decision actually **is** ridiculous.
+
+Take a read of our post on [Chesteron's Fence](./being-a-good-community-member/chestertons-fence) to practical examples of how this principle applies.
+
+## Disagree constructively
+
+A huge part of building a successful product is in people bringing different opinions and discussing them to aim to find truth (or at the very least some constructive way forward).
```

Comment:
```suggestion
A huge part of building a successful product and a successful community centers on people bringing in different opinions and perspectives and discussing those with the aim of finding truth (or at the very least some constructive way forward).
```

Reply:
```suggestion
A huge part of building a successful product and a successful community centers on people bringing in different opinions and perspectives and discussing those with the aim of finding a common truth. It may even require some disagreement about the accepted truth to arrive at a better truth.
```